templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,34 @@
+# Import your settings
+
+When you create a new account using an email address already associated with a
+Zulip account in another organization, the registration form will offer to import
+the user settings from one of your existing account profiles.
+
+You can only import personal settings (i.e., you will not be able to import
+organization-specific data such as stream settings or custom profile fields).
+
+The import will include your:
+
+- [Name and avatar]({{ root_domain_uri }}/#settings/profile)
+- [Privacy settings]({{ root_domain_uri }}/#settings/account-and-privacy)
+- [Display settings]({{ root_domain_uri }}/#settings/display-settings)
+- [Notification settings]({{ root_domain_uri }}/#settings/notifications)
+- Hotspots/tutorial state
+
+### Import your settings
+
+{start_tabs}
+
+1. [Review your settings](/help/review-your-settings) before
+importing them (recommended).
```

Comment:
What do you think about moving "recommended" to the front, like this? Let's also rephrase a bit.

> *(recommended)* [Review](/help/review-your-settings) the settings you plan to import.